Jenkinsfile

```diff
@@ -1,5 +1,10 @@
-mdsdToolsEclipsePipeline {  
-    webserverDir = 'thirdparty-library'
-    updateSiteLocation = 'target/repository'
-    skipCodeQuality = true
-} 
\ No newline at end of file
+@Library('MDSDtoolsLib@master') _
+MDSDToolsPipeline {
+    buildWithGradle {
+        propertiesId = '2d2bf442-7fac-4b6d-9305-1214ab640234'
+        jdkVersion = '11'
+    }
+    deployUpdatesite 'build/updatesite'
+    skipQualityMetrics true
+    skipDeploy false
```

Comment:
We should remove this flag before merging because otherwise, a branch of the master branch automatically creates a new folder on the update site.

Reply:
True.